### PR TITLE
2 readme

### PR DESCRIPTION
**Pull request by milanwiedemann**
what you two think about these naming guidelines for our functions across different packages?

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -28,27 +28,25 @@ You can install the released version of experienceAnalysis from [CRAN](https://C
...
-This is a basic example which shows you how to solve a common problem:
+### Data manipulations
 
-```{r example}
-library(experienceAnalysis)
-## basic example code
-```
+- `get_*()`: Get data, e.g., from a database or somewhere else
+- `tidy_*()`: Tidy data (generally from a database), e.g., renaming variables, removing duplicates, creating factors, 'wide' to 'long' format
+- `collect_*()`: Collect data of specific cases, mainly wrapper functions for specific filter commands
```
Comment (andreassot10) [inline README.Rmd:37]: Can you please explain this a bit better.

Comment (milanwiedemann) [reply README.Rmd:37]: of course, I will try

File: README.Rmd [hunk 2]
```diff
@@ -28,27 +28,25 @@ You can install the released version of experienceAnalysis from [CRAN](https://C
...
 
-This is a basic example which shows you how to solve a common problem:
+### Data manipulations
 
-```{r example}
-library(experienceAnalysis)
-## basic example code
-```
+- `get_*()`: Get data, e.g., from a database or somewhere else
+- `tidy_*()`: Tidy data (generally from a database), e.g., renaming variables, removing duplicates, creating factors, 'wide' to 'long' format
```
Comment (andreassot10) [inline README.Rmd:36]: Please remove "," after "e.g.". Also, I don't think "(generally from a database)" is necessary here.

**Comment by ChrisBeeley** [comment]
I think it looks good. I know I always say this but I think we really need to write some code before we get a feel for if it's all correct. I'm doing a lot of work on {experiencesdashboard} over the next few weeks so that will help me to understand what we need and what is a logical name for it

**Pull request merged**